dandiapi/zarr/views/__init__.py

```diff
@@ -97,13 +94,11 @@ class Meta:
     upload_in_progress = serializers.BooleanField(source='uploads_exist')
 
 
-class ZarrExploreSerializer(serializers.Serializer):
-    directories = serializers.ListField(child=serializers.URLField())
-    files = serializers.ListField(child=serializers.URLField())
-    checksums = serializers.DictField(
-        child=serializers.RegexField('^[0-9a-f]{32}(-[0-9]+--[0-9]+)?$')
-    )
-    checksum = serializers.RegexField('^[0-9a-f]{32}-[0-9]+--[0-9]+$')
+class ZarrExploreQuerySerializer(serializers.Serializer):
+    after = serializers.CharField(default='')
+    prefix = serializers.CharField(default='')
+    limit = serializers.IntegerField(default=1000)
```

Comment:
What happens if the client provides an extremely large value or a negative value here? Do we need `max_value` and `min_value`?